counterparty-core/counterpartycore/lib/cli/main.py

```diff
@@ -104,6 +104,13 @@ def float_range_checker(arg):
         ("--backend-password",),
         {"default": "rpc", "help": "the password used to communicate with backend"},
     ],
+    [
```

Comment:
I think we should officially deprecate the backend user and password (but leave in support for a while).

Reply:
done!